app/assets/javascripts/search.js

```diff
@@ -16,9 +16,9 @@ function RealtimeStatus( id ) {
   });
 }
 
-function RealtimeItem( an , oclc ) {
+function RealtimeItem( an , oclc, scan ) {
   $.ajax({
-    url: "/full_item_status?id=" + an + '&oclc=' + oclc
+    url: "/full_item_status?id=" + an + '&oclc=' + oclc + '&scan=' + scan
```

Comment:
Are we worried that our students will simply edit the URLs when they don't see the 'request scan' button but want to? (I mean, I'm 100% confident this will happen, given this student population, with this implementation. I don't know if it'll happen often enough that we need to care.)

I'm also increasingly uncomfortable about how we're using this URL to couple Aleph and EDS data sources more and more tightly. I didn't like having to incorporate the OCLC number to make it work and now there's yet another parameter. Can we get this data from the Aleph side?

Reply:
I 100% don't think this is worth worrying about. I totally get it, but if someone really wants to request a scan and then be denied by staff they can indeed do so.

Reply:
Will update in-app documentation to explain why we are doing this mess.

Reply:
I've added a commit to explain the design decision in the AlephItem class.